chandra_models/xija/pline/pline03t_model_spec.json

```diff
@@ -156,6 +156,15 @@
     "datestop": "2020:001:11:54:30.816",
     "dt": 328.0,
     "evolve_method": 2,
+    "limits": {
+        "pline03t": {
+            "planning.warning.high": 200,
+            "planning.warning.high.matlab_identifier": "yellow_limit",
```

Comment:
similar to the MUPS models, should this be `planning.caution.high` as a yellow limit?

Reply:
Mostly the MATLAB `yellow_limit` corresponds to the planning limit (`planning.warning.high`) from what I could see.